packages/vuetify/src/composables/__tests__/position.spec.ts

```diff
@@ -0,0 +1,32 @@
+// Utilities
+import { usePosition } from '..'
+
+// Types
+import type { PositionProps } from '..'
+
+describe('position.ts', () => {
+  it.each([
+    [{ position: undefined }, []],
+    [{ position: false }, []],
+    [{ position: null }, []],
+    [{ position: 'absolute' }, ['position-absolute']],
+    [{ position: 'fixed' }, ['position-fixed']],
+  ])('should have proper classes', (props, expected) => {
+    const { positionClasses } = usePosition(props as PositionProps)
```

Comment:
i guess casting is here so that `position: null` doesn't throw ts error, but since `null` is not a valid value of prop i don't think testing it (and casting) is necessary